extensions/validation/src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -149,22 +156,17 @@ public void isFalse() throws Exception
     public void notEmpty() throws Exception
     {
         final String flowName = "notEmpty";
-
```

Comment:
We should agree on a way for doing this and stick to it always or leave the decision to the committer.